libmamba/src/api/configuration.cpp

```diff
@@ -1350,7 +1363,6 @@ namespace mamba
         insert(Configurable("envs_dirs", &m_context.envs_dirs)
                    .group("Basic")
                    .set_rc_configurable(RCConfigLevel::kHomeDir)
-                   .set_env_var_names({ "CONDA_ENVS_DIRS" })
```

Comment:
I know it seems weird to take this out, but this Configurable deserializes the value of the env var assuming it's YAML (e.g. comma-delimited list).  It's delimited with the os path separator instead, so we deal with that special case in the hook.